helper/chart_data.js

```diff
@@ -0,0 +1,2788 @@
+
+const path = require("path");
+
+const default_no_of_assessment_submissions_threshold = 3;
+const default_entity_score_api_threshold = 5;
+const filesHelper = require(MODULES_BASE_PATH + '/cloud-services/files/helper')
+
+//function for instance observation final response creation
+exports.instanceReportChart = async function (data, reportType = "") {
```

Comment:
@borkarsaish65 Please add function description where ever it is missing

Reply:
done